client/landing/stepper/declarative-flow/connect-domain.ts

```diff
@@ -101,18 +101,23 @@ const connectDomain: Flow = {
 			},
 		];
 	},
+	useTracksEventProps( event ) {
+		const { domain, provider } = useDomainParams();
+
+		if ( STEPPER_TRACKS_EVENT_STEP_NAV_SUBMIT === event ) {
+			return {
+				domain,
+				provider,
+			};
+		}
```

Comment:
These can be queried in `useStepNavigationWithTracking`. Added here primarily for testing, as the only case in the code where additional props are passed to `recordSubmitStep`.